test/e2e/rolling_update_test.go

```diff
@@ -0,0 +1,297 @@
+// (C) Copyright Confidential Containers Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/e2e-framework/klient"
+	"sigs.k8s.io/e2e-framework/klient/k8s"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	envconf "sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+const WAIT_DEPLOYMENT_AVAILABLE_TIMEOUT = time.Second * 180
+
+func doTestCaaDaemonsetRollingUpdate(t *testing.T) {
+	runtimeClassName := "kata-remote"
+	namespace := envconf.RandomName("default", 7)
+	deploymentName := "nginx-deployment"
+	containerName := "nginx"
+	imageName := "nginx"
+	serviceName := "nginx-service"
+	portName := "port80"
+	rc := int32(2)
+	labelsMap := map[string]string{
+		"app": "nginx",
+	}
+	verifyPodName := "verify-pod"
+	verifyContainerName := "verify-container"
+	verifyImageName := "radial/busyboxplus:curl"
```

Comment:
can we use mulit-arch verify image here? so that we can run this case with s390x clusters.
it seems that `radial/busyboxplus:curl` only have amd64 docker image